generator-theia/src/extension/extension-package-generator.ts

```diff
@@ -101,4 +104,26 @@ export class ExtensionPackageGenerator extends AbstractGenerator {
         }
     }
 
-}
\ No newline at end of file
+    protected compilePackageTdOptions(): object {
+        return {
+            "excludeNotExported": true,
+            "exclude": "lib, node_modules",
+            "experimentalDecorators": true,
+            "ignoreCompilerErrors": true,
+            "mode": "file",
+            "out": "./docs/" + this.getPackageName(),
```

Comment:
But actually there's no need for the package name like I said before you are already in the package.... use docs/api